src/record/mutation.ts

```diff
@@ -373,12 +390,41 @@ export default class MutationBuffer {
           };
           this.attributes.push(item);
         }
-        // overwrite attribute if the mutations was triggered in same time
-        item.attributes[m.attributeName!] = transformAttribute(
-          document,
-          m.attributeName!,
-          value!,
-        );
+        if (m.attributeName === 'style') {
+          const old = document.createElement('span');
+          old.setAttribute('style', m.oldValue);
+          if (item.attributes['style'] === undefined) {
+            item.attributes['style'] = {};
+          }
+          for (let i=0; i<target.style.length; i++) {
+            let pname = target.style[i];
+	    const new_value = target.style.getPropertyValue(pname);
+	    const new_priority = target.style.getPropertyPriority(pname);
```

Comment:
For code hygiene it's probably better to camelCase these variable names. 
```suggestion
	    const newValue = target.style.getPropertyValue(pname);
	    const newPriority = target.style.getPropertyPriority(pname);
```

Reply:
Thanks; I take it I cannot just commit this suggestion as-is?  
I'll make a new commit.

Reply:
Updated the original commit with your suggested change as was force-pushing anyhow.